xbmc/cores/VideoPlayer/Process/ProcessInfo.h

```diff
@@ -62,7 +62,7 @@ class CProcessInfo
   bool Supports(EINTERLACEMETHOD method);
   void SetDeinterlacingMethodDefault(EINTERLACEMETHOD method);
   EINTERLACEMETHOD GetDeinterlacingMethodDefault();
-  CVideoBufferManager& GetVideoBufferManager();
+  virtual CVideoBufferManager& GetVideoBufferManager();
```

Comment:
it should not be necessary to override the manager. the manager provices methods for pltforms to register their pools